### PR TITLE
ask confirmation before importing gpg key

### PR DESCRIPTION
**Pull request by mbussolotto**
## What does this PR change?

ask confirmation before importing gpg key

## Test coverage
- No tests
- [ ] **DONE**

## Links

Issue(s): #

- [ ] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

**Comment by mbussolotto** [comment]
if you want to have a look at the library I used https://github.com/manifoldco/promptui.

File: shared/utils/utils.go [hunk 1]
```diff
@@ -88,6 +90,19 @@ func AskIfMissing(value *string, prompt string, min int, max int, checker func(s
 	}
 }
 
+// YesNo asks a question in CLI.
+func YesNo() (bool, error) {
+	prompt := promptui.Select{
+		Label: "Select[Yes/No]",
+		Items: []string{"Yes", "No"},
+	}
+	_, result, err := prompt.Run()
```
Comment (cbosdo) [inline shared/utils/utils.go:99]: Do we need a prompt library and its deps for this? If yes, then we'll need to standardize on it. I have no clue how to pick one Go library of the other... @rjmateus any opinion?

File: mgradm/cmd/gpg/add/gpg.go [hunk 2]
```diff
@@ -43,7 +43,7 @@ func NewCommand(globalFlags *types.GlobalFlags) *cobra.Command {
 		},
 	}
 
-	gpgAddKeyCmd.Flags().BoolP("force", "f", false, L("Run the import"))
+	gpgAddKeyCmd.Flags().BoolP("force", "f", false, L("force import without asking confirmation"))
```
Comment (cbosdo) [inline mgradm/cmd/gpg/add/gpg.go:46]: We'll need to harmonize the case of the flags help to start with lowercase at some point...

File: shared/utils/utils.go (hunk 1 above)
Comment (rjmateus) [reply shared/utils/utils.go:99]: We are already implementing interactivity to get the password for the database, for example. We should use the same method as before and avoid adding one more dependency just for this.

**Pull request merged**